CHANGELOG.md

```diff
@@ -2,6 +2,73 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.0] - 2019-07-??
+ - New Features
+   - `pysatData` directory created in user's home directory if no directory specified
+   - Added preliminary support for `xarray` to the `instrument` object
+   - Support for `today`, `tomorrow`, and `yesterday` as datetime objects
+   - Added `model_utils`, featuring preliminary support for data-model comparison
+   - Added support for 1d median in seasonal averages
+   - Added routine to convert from kp to Ap
+   - Added `pyglow` integration support for python 3.x
+   - Improves input handling for datetime parameters that are more precise than just year, month, and day, where appropriate
+   - Added merging routines to allow combination of measured and forecasted Kp and F10.7 indexes into a single instrument object
+ - Code restructure
+   - Moved instrument templates and methods to subdirectories
+   - Moved utils into multiple subdirectories to aid with organization
+ - Instrument Updates
+   - NASA CDAWeb download now uses https protocol rather than FTP
+   - `_instrument.py` supports xarray
+   - Support for listing files from remote server
+   - COSMIC RO data unified into single instrument object
+   - Added support for DEMETER IAP
+   - Added support for DMSP IVM Level 1 data
+   - Added routines to update DMSP ephemeris and drifts
+   - Added warnings to instruments without download support
+   - Added support for ICON FUV and MIGHTI
+   - Added support for JRO ISR
+   - Added support for F10.7 and more Kp forecast products
+   - Added instrument templates for Madrigal, CDAWeb, and netcdf_pandas
+   - Added support for TIMED SABER
```

Comment:
omni_hro was also shifted to use cdaweb methods.

Reply:
fixed